src/main/webapp/app/exam/manage/student-exams/student-exams.component.html

```diff
@@ -23,12 +23,6 @@ <h2>
                     }
                     <span jhiTranslate="artemisApp.studentExams.generateMissingStudentExams"></span>
                 </button>
-                <button id="startExercisesButton" class="btn btn-primary mt-1" (click)="startExercises()" [disabled]="isLoading || isExamStarted || exercisePreparationRunning">
-                    @if (isLoading) {
-                        <span class="spinner-border spinner-border-sm" role="status" aria-hidden="true"></span>
-                    }
-                    <span jhiTranslate="artemisApp.studentExams.startExercises"></span>
```

Comment:
The i18n key `artemisApp.studentExams.startExercises` was only used here. Therefore, this key is not in usage anymore, is it? If so, it should be removed from the `exam.json` for both languages.

Reply:
done